apache/zeppelin
# [ZEPPELIN-1926] Remove grunt googlefonts task

### What is this PR for?

There is no reason to download fonts files whenever we build.

- Google font doesn't change frequently.
- We can use google fonts in open source (https://developers.google.com/fonts/faq)

> Yes. The open source fonts in the Google Fonts catalog are published under licenses that allow you to use them on any website, whether it’s commercial or personal. 

For reviewer, please review `Gruntfile.js` and `package.json`. Other 49 file changes are just about adding font resources.


### What type of PR is it?
[Improvement]

### Todos

Fixed at once

### What is the Jira issue?

[ZEPPELIN-1926](https://issues.apache.org/jira/browse/ZEPPELIN-1926)

### How should this be tested?

1. Run zeppelin-web dev mode
2. Open browser and check 404

### Screenshots (if appropriate)

None

### Questions:
* Does the licenses files need update? - NO
* Is there breaking changes for older versions? - NO
* Does this needs documentation? - NO
